### ISSUE TITLE
Prefix error messages with "Error:"

### ISSUE THREAD

**Issue by jeanniefu** [civiform/civiform#3286]
From our accessibility advisors - Accessibility criteria: "Color is not used as the sole means of indicating the error message".

Currently, we display error messages in red without another indication that is is an error. This does not work well for users who are color-blind. We should include the text "Error" before the message.

**Comment by melanie-exygy** [comment]
@sijiayam I'm working on this as a first issue.  Do you think this applies to toast error messages, error pages or both?  Right now toast errors look like this:  
![image](https://github.com/civiform/civiform/assets/134556723/c3a7fad8-3ff3-4302-87b5-936387a7c4f4)
and error pages look like this:  
![image](https://github.com/civiform/civiform/assets/134556723/3eef9f55-8036-4a06-bc42-069ada14d891)
